hpx/parallel/execution_policy.hpp

```diff
@@ -1418,6 +1423,11 @@ namespace hpx { namespace parallel { HPX_INLINE_NAMESPACE(v1)
         template <>
         struct is_parallel_execution_policy<parallel_task_execution_policy>
           : std::true_type
+        {};        
+
+        template <typename Executor, typename Parameters>
+        struct is_parallel_execution_policy<parallel_task_execution_policy_shim<Executor, Parameters>>
+          : std::true_type
         {};
         /// \endcond
```

Comment:
The `parallel_task_...` policies are 'parallel' as well. Could you add those while you're at it as well, please?